kernel/type.h

```diff
@@ -31,6 +31,7 @@
 #define FALSE 0
 #define NOT_FOUND -1
 #define MAX_STRING_LENGTH 100000
+#define BiConsumer std::function
```

Comment:
I think we should comment this and wait for BiConsumer class

Reply:
Please remove this definition because it's wrong. Should be BiConsumer implementation

Reply:
Thanks. Fixed.